resources/usa_lottery/server/sv_lottery.lua

```diff
@@ -104,3 +113,23 @@ RegisterServerCallback {
         end
     end
 }
+
+function SaveNewBalance(finishedBalance, fundAccount)
+    TriggerEvent('es:exposeDBFunctions', function(db)
+        db.updateDocument("govfunds", fundAccount, { content = finishedBalance }, function(doc, err, rText)
```

Comment:
```suggestion
        exports.essentialmode:updateDocument("govfunds", fundAccount, { content = finishedBalance })
```

And also get rid of the `es:exposeDBfunctions` event part